docker/root/etc/s6/gitea/setup

```diff
@@ -44,6 +44,8 @@ if [ ! -f ${GITEA_CUSTOM}/conf/app.ini ]; then
     SECRET_KEY=${SECRET_KEY:-""} \
     envsubst < /etc/templates/app.ini > ${GITEA_CUSTOM}/conf/app.ini
```

Comment:
Can't we drop those lines and for example do ?
```
SECRET_KEY=${SECRET_KEY:-""}
GITEA__security__SECRET_KEY=${GITEA__security__SECRET_KEY:-"$SECRET_KEY"}
```
(maybe they can get in one line)
The goal could be to deprecate the old variables and use the new generic only later ? 
So that `${GITEA_CUSTOM}/conf/app.ini` is only write once and drop template `app.ini`